examples/DQMC/from_fcidump/h2o/h2o.py

```diff
@@ -0,0 +1,58 @@
+import sys, os
+import numpy
+import numpy as np
+import QMCUtils
```

Comment:
probably don't want to import QMCUtils because it imports pyscf modules, maybe copy the required functions at the top of this script